src/components/text.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+import styled from "styled-components"
+import * as fonts from "../assets/fonts"
+
+type TextAligns = "start" | "center" | "end"
+type TextSizes = "small" | "large"
+
+interface TextProps extends React.HTMLProps<JSX.Element> {
+  textSize?: TextSizes,
+  align?: TextAligns,
+  color?: string,
+}
+
+const textSizes = {
+  small: "17px",
+  large: "20px",
+}
+
+const Text: React.SFC<TextProps> = props => (
+  <div className={props.className}>
+    {props.children}
+  </div>
```

Comment:
I have a slight preference towards `p`, because I’d assume that assisted reading methods make use of these semantic differences. However, I have not looked into it, so maybe my assumption is incorrect?